distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc

```diff
@@ -0,0 +1,196 @@
++
+[source,console]
+----
+$ oc edit configmap tempo-operator-manager-config -n openshift-tempo-operator <1>
+----
+<1> The project where the {TempoOperator} is installed.
+
+. Disable the mTLS in the Operator configuration.
++
+[source,yaml]
+----
+data:
+  controller_manager_config.yaml: |
+    featureGates:
+      httpEncryption: false
+      grpcEncryption: false
+      builtInCertManagement:
+        enabled: false
+----
+
+. Restart the {TempoOperator} Pod.
++
+[source,console]
+----
+$ oc rollout restart deployment.apps/tempo-operator-controller -n openshift-tempo-operator
+----
+====
+
+* Missing images for running the {TempoOperator} in restricted environments. The {TempoName} CSV is missing references to the operand images. (link:https://issues.redhat.com/browse/TRACING-3173[TRACING-3523])
++
+.Workaround
+:!example-caption:
+====
+
+Add the {TempoOperator} related images in the mirroring tool to mirror the images to the registry:
+
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+archiveSize: 20
+storageConfig:
+  local:
+    path: /home/user/images
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13
+    packages:
+    - name: tempo-product
+      channels:
+      - name: stable
+  additionalImages:
+  - name: registry.redhat.io/rhosdt/tempo-rhel8@sha256:e4295f837066efb05bcc5897f31eb2bdbd81684a8c59d6f9498dd3590c62c12a
+  - name: registry.redhat.io/rhosdt/tempo-gateway-rhel8@sha256:b62f5cedfeb5907b638f14ca6aaeea50f41642980a8a6f87b7061e88d90fac23
+  - name: registry.redhat.io/rhosdt/tempo-gateway-opa-rhel8@sha256:8cd134deca47d6817b26566e272e6c3f75367653d589f5c90855c59b2fab01e9
+  - name: registry.redhat.io/rhosdt/tempo-query-rhel8@sha256:0da43034f440b8258a48a0697ba643b5643d48b615cdb882ac7f4f1f80aad08e
+----
+
+====
+
+== {OTELName}
+
+:FeatureName: The {OTELName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+=== New features and enhancements
+
+This release introduces the following noteworthy enhancements for the {OTELShortName}:
+
+* Support OTLP metrics ingestion. The metrics can be forwarded and stored in the `user-workload-monitoring` via the Prometheus exporter.
+
+* Support the link:https://operatorframework.io/operator-capabilities/[operator maturity] Level IV, Deep Insights, which enables upgrading and monitoring of `OpenTelemetry Collector` instances and the {OTELOperator}.
+
+* Report traces and metrics from remote clusters using OTLP/HTTP(s).
+
+* Collect {product-title} resource attributes via the `resourcedetection` processor.
+
+* Support managed and unmanaged states in the `OpenTelemetryCollector` custom resouce.
+
+//=== Technology Preview features
+//not for 2.9
+
+=== Bug fixes
+
+* Before this update, the {OTELOperator} was crashlooping after receiving `opentelemetry-operator.v0.74.0-5`. With this update, the issue is fixed by configuring the autoscaler. (link:https://issues.redhat.com/browse/TRACING-3190[TRACING-3190])
```

Comment:
Should `autoscaler` be `cluster autoscaler`?
```suggestion
* Before this update, the {OTELOperator} was crashlooping after receiving `opentelemetry-operator.v0.74.0-5`. With this update, the issue is fixed by configuring the autoscaler. (link:https://issues.redhat.com/browse/TRACING-3190[TRACING-3190])
```

Reply:
@iblancasa, is this about configuring the horizontal pod autoscaler?

```suggestion
* Before this update, the {OTELOperator} was crashlooping after receiving `opentelemetry-operator.v0.74.0-5`. With this update, the issue is fixed by configuration of the horizontal pod autoscaler. (link:https://issues.redhat.com/browse/TRACING-3190[TRACING-3190])
```

Reply:
There is a description of the problem here https://github.com/open-telemetry/opentelemetry-operator/issues/1799

The HPA  can be configured using the  `.spec.minReplicas` and `.spec.maxReplicas` fields (those are deprecated). It should be done using the fields under `.spec.autoscaler`. During one refactor of the HPA logic, the `.spec.minReplicas` and `.spec.maxReplicas` fields were not taken into account, making the upgrade from one version to another to fail.

Reply:
Thank you for the suggestion.
Luckily, now that we've looked more in-depth into this release note, we can drop it.
Removing it.